src/AspNet.Security.OAuth.DingTalk/DingTalkAuthenticationOptions.cs

```diff
@@ -35,7 +35,11 @@ public DingTalkAuthenticationOptions()
             ClaimActions.MapJsonKey(Claims.DingId, "dingId");
             ClaimActions.MapJsonKey(Claims.MainOrgAuthHighLevel, "main_org_auth_high_level");
         }
-
+        public bool IsUseUserPasswordLogin
```

Comment:
I don't really like this - it's not a good name, the reason for its existence isn't clear, the get value only returns `true` for one possible exact string, even if there's some other URL set that is also username-password authentication.

Reply:
Dingtalk code scanning login and password login cannot be switched online, so this "IsUseUserPasswordLogin" option is made

# QRConnect
![image](https://user-images.githubusercontent.com/1501040/71757270-607af400-2ecf-11ea-9e6d-de2979f231c3.png)

# Password
![image](https://user-images.githubusercontent.com/1501040/71757325-dbdca580-2ecf-11ea-9927-3212b8930e02.png)

![image](https://user-images.githubusercontent.com/1501040/71757328-e008c300-2ecf-11ea-9151-e53c5b796f53.png)